common/remote/hooks/git/gitfile.cpp

```diff
@@ -139,6 +139,10 @@ class GitRepositoryFileIO : public CInterface, implements IFileIO
     {
         throwUnexpected();
     }
+    unsigned __int64 getStatistic(StatisticKind kind)
+    {
+        return 0;
```

Comment:
Is this because they are meaningless, or because you are lazy? I would have thought the stats were valid, though we tend not to read these files from ECL jobs...